.github/workflows/lint_test_charts.yaml

```diff
@@ -141,3 +141,34 @@ jobs:
           cut -d\  -f2 versions.txt | sort -u
 
           [ $(cut -d\  -f2 versions.txt | sort -u | wc -l) -eq 1 ] || exit 1
+
+  bundle-dependencies-check:
+    name: Test the bundle dependency conditions
+    runs-on: ubuntu-latest
+    if: ${{ needs.lint-test.outputs.bundle-changed == 'true' }}
+    needs: [ lint-test ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: azure/setup-helm@v3
+        with:
+          version: 'v3.0.0'
+
+      - name: Add helm repositories
+        run: |
+          helm repo add newrelic-helm-charts https://newrelic.github.io/helm-charts
+          helm repo add newrelic-cdn-helm-charts https://helm-charts.newrelic.com
+          helm repo add newrelic-infrastructure https://newrelic.github.io/nri-kubernetes
+          helm repo add nri-prometheus https://newrelic.github.io/nri-prometheus
+          helm repo add newrelic-prometheus-configurator https://newrelic.github.io/newrelic-prometheus-configurator
+          helm repo add nri-metadata-injection https://newrelic.github.io/k8s-metadata-injection
+          helm repo add newrelic-k8s-metrics-adapter https://newrelic.github.io/newrelic-k8s-metrics-adapter
+          helm repo add kube-state-metrics https://kubernetes.github.io/kube-state-metrics
+          helm repo add nri-kube-events https://newrelic.github.io/nri-kube-events
+          helm repo add pixie-operator-chart https://pixie-operator-charts.storage.googleapis.com
+          helm repo add newrelic-infra-operator https://newrelic.github.io/newrelic-infra-operator
```

Comment:
It is a little bit more cryptic and we also need to install `yq` but I think it does totally worth it: the less places we are repeating the repos the better!

Really good suggestion! I will update this kind of repo add in the pipeline.

Reply:
The installing `yq` concern was only because I didn't know it is already in the ubuntu runners (I saw we install it in [other pipeline](https://github.com/newrelic/newrelic-prometheus-exporters-packages/blob/main/.github/workflows/prerelease.yml#L29) and I though we needed it (even so I already thought it does worth it).

Regarding being a little bit more cryptic, I think it is... But, as I see it, it is still easy to understand.